src/presenters/filter-presenter.js

```diff
@@ -0,0 +1,58 @@
+import {render, replace, remove} from '../framework/render.js';
+import FilterView from '../view/filter-view.js';
+import {filter} from '../utils.js';
+import {UpdateType} from '../constant.js';
+
+export default class FilterPresenter {
+  #filterContainer = null;
+  #filterModel = null;
+  #pointsModel = null;
+
+  #filterComponent = null;
+
+  constructor({filterContainer, filterModel, pointsModel}) {
+    this.#filterContainer = filterContainer;
+    this.#filterModel = filterModel;
+    this.#pointsModel = pointsModel;
+
+    this.#pointsModel.addObserver(this.#handleModelEvent);
+    this.#filterModel.addObserver(this.#handleModelEvent);
+  }
+
+  get filters() {
+    const points = this.#pointsModel.points;
+
+    return filter(points);
```

Comment:
```suggestion
    return getFilterData(points);
```
filter похож на метод массива, бывают снимают критерий на защите за это